src/common/meta/src/key/table_route.rs

```diff
@@ -38,42 +38,66 @@ impl TableRouteKey {
 }
 
 #[derive(Debug, PartialEq, Serialize, Deserialize, Clone)]
-pub struct TableRouteValue {
+pub enum TableRouteValue {
+    Mito(MitoTableRouteValue),
+    Metrics(MetricsTableRouteValue),
+}
+
+#[derive(Debug, PartialEq, Serialize, Deserialize, Clone)]
+pub struct MitoTableRouteValue {
```

Comment:
`MitoTableRouteValue` looks not good. The file table also uses this type of route. cc @evenyag @waynexia

Reply:
Maybe file engine table should have its own `FileTableRouteValue`.

Reply:
I suspect if it is necessary to make variants over engine type...

Reply:
I think the route value type should not be determined by the engine type. There should only be two types: real and virtual.